index.html

```diff
@@ -0,0 +1,539 @@
+      <h4>By <span>Sandamal Siripathi</span></h4>
+      <p>Transliteration is a popular way to input non-English characters using an English keyboard. This article
+        explains a new method to eliminate such browser-related problems when developing web apps that use
+        transliteration.</p>
+      <div class="article-arrow">→</div>
+    </div>
+    <div>
+      <h5>5 days ago</h5>
+      <h1>Avoid keyboard event-related box in browser-based transliteration</h1>
+      <h4>By <span>Sandamal Siripathi</span></h4>
+      <p>Transliteration is a popular way to input non-English characters using an English keyboard. This article
+        explains a new method to eliminate such browser-related problems when developing web apps that use
+        transliteration.</p>
+      <div class="article-arrow">→</div>
+    </div>
+    <div>
+      <h5>5 days ago</h5>
+      <h1>Avoid keyboard event-related box in browser-based transliteration</h1>
+      <h4>By <span>Sandamal Siripathi</span></h4>
+      <p>Transliteration is a popular way to input non-English characters using an English keyboard. This article
+        explains a new method to eliminate such browser-related problems when developing web apps that use
+        transliteration.</p>
+      <div class="article-arrow">→</div>
+    </div>
+    <div>
+      <h5>5 days ago</h5>
+      <h1>Avoid keyboard event-related box in browser-based transliteration</h1>
+      <h4>By <span>Sandamal Siripathi</span></h4>
+      <p>Transliteration is a popular way to input non-English characters using an English keyboard. This article
+        explains a new method to eliminate such browser-related problems when developing web apps that use
+        transliteration.</p>
+      <div class="article-arrow">→</div>
+    </div>
+    <div class="advert">
+      <div class="advert-div">
+        <img src="images/cat-smashing-tv-2.svg" alt="">
+        <h1>New on smashing TV</h1>
+        <p>A channel where seasoned developers show how they design, build, work and play. Coming up next: <a
+            href="#">Building A Modern Electron App With React Hooks</a> with Cassidy Williams (Mar 19).</p>
+        <h4> See all upcoming episodes →</h4>
+        <h5>Already have an account? <a href="#">Sign in!</a></h5>
+      </div>
+    </div>
+    <p class="see-all">See all latest articles →</p>
+  </article>
+  <section class="ethical">
+    <div>
+      <h5>March 2020 • Shipping now</h5>
+      <h1><a href=""> Ethical Design Handbook:</a></h1>
+      <h2>Leaving Dark Patterns Behind.</h2>
+      <p>With <strong> practical techniques & strategies</strong> to make honest design work for digital products.
+        Help your business grow sustainably — without dark patterns — and comply with GDPR and CCPA.</p>
+      <h4>Learn more about the book →</h4>
+      <img src="images/ethical-design-transparent.png" alt="Ethical Deisgn">
+    </div>
+  </section>
+  <section class="community">
```

Comment:
Get rid of the image and the text, just highlight the elements inside this section

Reply:
Done

Reply:
Done!